src/MSBuild/XMake.cs

```diff
@@ -1788,6 +1790,10 @@ private static void GatherResponseFileSwitch(string unquotedCommandLineArg, Comm
                                 // skip comment lines beginning with #
                                 if (!responseFileLine.StartsWith("#", StringComparison.Ordinal))
                                 {
+                                    // Allow special case to support a path relative to the .rsp file being processed.
+                                    responseFileLine = Regex.Replace(responseFileLine, responseFilePathReplacement,
```

Comment:
Should we be worried perfwise that we are running this regex for every line in the rsp file?

Reply:
Most of the files are tiny and read anyway. On my machine I had to make the file like 4000 lines before I saw a 1ms overhead for the regex, and they're typically 1-5 lines or so.